helpers/compile/build.ts

```diff
@@ -36,51 +34,23 @@ const DEFAULT_BUILD_OPTIONS = {
  * Apply defaults to allow us to build tree-shaken esm
  * @param options the original build options
  */
-const applyEsmDefaults = (options: BuildOptions): BuildOptions => ({
```

Comment:
What's the short human version of this change? 😄

Reply:
Basically, we were doing a hack of compiling TypeScript-commonjs-ish code to esm so that it can get tree-shaken, then that code would be transformed back to cjs. It was a hack that worked until [esbuild 0.14.4/5](https://github.com/evanw/esbuild/releases/tag/v0.14.4). I happily got rid of the hack to use TS `paths` instead.

Reply:
Indeed! Very nice 👍🏼